skimage/data/__init__.py

```diff
@@ -117,7 +117,7 @@ def create_image_fetcher():
     # This helps pooch understand that it should look in master
     # to find the required files
     pooch_version = __version__.replace('.dev', '+')
-    url = "https://github.com/scikit-image/scikit-image/raw/{version}/skimage/"
```

Comment:
Does this need changing before merging @alexdesiqueira?

Reply:
It does @jni, we'll get back to the original one when the PR is ready :slightly_smiling_face:

Reply:
We need an update here :)